scipy/linalg/tests/test_lapack.py

```diff
@@ -1826,6 +1821,9 @@ def test_gejsv_general(size, dtype, joba, jobu, jobv, jobr, jobt, jobp):
     Although all arguments are tested, the tests only check that the correct
     solution is returned - NOT that the prescribed actions are performed
     internally.
+
+    jobt is, as of v3.9.0, still experimental and removed to cut down number of
+    test cases. However keyword itself is tested externally.
```

Comment:
✓
By externally, do you mean we have tested it before - it's just not being included here?

Reply:
Yes the keyword works and we shouldn't be responsible about its function. As long as it is experimental, it's probably better that we don't touch it anyways.